usr/share/rear/verify/BAREOS/default/100_bextract_mount.sh

```diff
@@ -0,0 +1,15 @@
+#
+# mount backup device
+#
+
+if [ "$BEXTRACT_DEVICE" ] || [ "$BEXTRACT_VOLUME" ]; then
```

Comment:
Shouldn't that check for the bareos restore mode (bconsole or bextract)?

Reply:
adapted